package.json

```diff
@@ -81,7 +81,7 @@
     "@azure/core-paging": "^1.6.2",
     "@eslint/compat": "^1.1.0",
     "@eslint/js": "^9.5.0",
-    "@fluentui/react-components": "^9.54.13",
+    "@fluentui/react-components": "^9.55.1",
```

Comment:
Just curious, does the update has anything to do with the fix?

Reply:
Nope.  I updated the package in case that was the explanation for the difference between windows and mac.  It wasn't, but I felt like it worth keeping.